Ladybird/WebDriver/main.cpp

```diff
@@ -33,6 +35,25 @@ static char** environment()
 #endif
 }
 
+static ErrorOr<pid_t> launch_process(StringView application, char const* argv[])
+{
+    // FIXME: Why can't we TRY() an ErrorOr<T> in a function returning ErrorOr<pid_t>?
```

Comment:
hmm what error did you get here? I pulled this locally and was able to build with:
```diff
-    // FIXME: Why can't we TRY() an ErrorOr<T> in a function returning ErrorOr<pid_t>?
-    auto paths_or_error = get_paths_for_helper_process(application);
-    if (paths_or_error.is_error())
-        return paths_or_error.release_error();
-    auto paths = paths_or_error.release_value();
+    auto paths = TRY(get_paths_for_helper_process(application));
```

Reply:
... huh. It was complaining about no constructor to convert b/w the TRY'd type and `ErrorOr<pid_t>`. But you're right, that works now. ... I am confused but happy. I'll push that then.